runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftConfig.java

```diff
@@ -37,19 +37,32 @@
 public interface RunecraftConfig extends Config
 {
 	@ConfigItem(
-		keyName = "showPouch",
+		keyName = "overlayStyle",
+		name = "Color overlay style",
+		description = "Configures the colored overlay style",
+		position = 0
+	)
+	default OverlayStyle overlayStyle()
+	{
+		return OverlayStyle.OUTLINE;
+	}
+
+	@ConfigItem(
+		keyName = "showPouchCount",
```

Comment:
Changing the key name will forget all existing settings people have, are you sure you want that?

Reply:
Whoops, that was probably a find/replace issue. Granted, the behavior is technically different. But I'll change it back.